docs/deprecated_doc/Alica_howto_get_the_framework_running.md

```diff
@@ -27,7 +27,7 @@
 	rosdep update
 	echo "source /opt/ros/indigo/setup.bash" >> ~/.bashrc
 	. ~/.bashrc
-	sudo apt-get install python-rosinstall
+	sudo apt-get install python3-rosinstall
```

Comment:
If this is deprecated maybe we just remove it? @veerajsk

Reply:
not sure about this

Reply:
>Install Ubuntu 14.04 LTS with a partition of at least 60 GB.

This is ancient so I think we should just get rid of it